node/blockchain/serializers/signed_change_request.py

```diff
@@ -11,8 +11,17 @@ class SignedChangeRequestSerializer(serializers.Serializer, ValidateUnknownField
     signature = serializers.CharField()
     message = serializers.JSONField()
 
+    class Meta:
+        dataclass = SignedChangeRequest
```

Comment:
it is not a dataclass, but a pydantic model

Reply:
Removed it be course of "SignedChangeRequest.parse_obj" method.